api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -13,33 +13,13 @@
 import static org.openmrs.Order.Action.REVISE;
 
 import java.lang.reflect.Field;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Calendar;
-import java.util.Collections;
-import java.util.Date;
-import java.util.List;
-import java.util.Locale;
-import java.util.Vector;
+import java.util.*;
```

Comment:
Please change this IntelliJ setting. We don't do any wildcard imports in non-test code.